EKF/optflow_fusion.cpp

```diff
@@ -428,16 +266,56 @@ void Ekf::fuseOptFlow()
 
 	}
 
-	_fault_status.flags.bad_optflow_X = false;
-	_fault_status.flags.bad_optflow_Y = false;
-
+	// fuse observation axes sequentially
 	for (uint8_t obs_index = 0; obs_index <= 1; obs_index++) {
 
-		// copy the Kalman gain vector for the axis we are fusing
-		Vector24f gain;
+		// calculate observation Jocobians and Kalman gains
+		if (obs_index == 0) {
+			// Observation Jacobians - axis 0
```

Comment:
Very nice, I really like that we can now compute K and H after computing the innovation variance of both axis. This is how it has been done everywhere else.